src/components/box/index.tsx

```diff
@@ -1,3 +1 @@
-import { Box, BoxProps } from '@chakra-ui/react';
-export { BoxProps };
-export default Box;
+export { Box, type BoxProps } from '@chakra-ui/react';
```

Comment:
Named export instead of default for better storybook code generation. 
#### Before
```jsx
<__WEBPACK_DEFAULT_EXPORT__>I am a box</__WEBPACK_DEFAULT_EXPORT__>
```

#### After
```jsx
<Box>I am a box</Box>
```

This doesn't impact all of the components but we want to apply the same pattern for consistency